### PR TITLE
Bump ort version number

### PR DESCRIPTION
**Pull request by RandySheriffH**
Bump ort version after the creation of release candidate of 1.12.

### CODE REVIEW

File: docs/Versioning.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.13.0 | **1.10** down to 1.2 | 15 | 2 | 7 | Windows AI 1.11+ |
 | 1.12.0 | **1.10** down to 1.2 | 15 | 2 | 7 | Windows AI 1.11+ |
```
Comment (RandySheriffH) [inline docs/Versioning.md:30]: Shouldn't be the opset version number be aligned with https://github.com/onnx/onnx/blob/main/docs/Versioning.md#released-versions

Comment (RandySheriffH) [reply docs/Versioning.md:30]: TODO - remove the table and point to https://github.com/microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix

**Comment by stonelazy** [comment]
When this gets merged to master, will we be able to use Opset 17 of ONNX ? [Reference](https://github.com/onnx/onnx/releases/tag/v1.12.0)

**Review by faxu — DISMISSED** [review]

**Review by faxu — DISMISSED** [review]

**Review by baijumeswani — APPROVED** [review]

**Comment by ytaous** [comment]
FYI - impacting ROCm docker iamge build - https://github.com/microsoft/onnxruntime/issues/12290

**Pull request merged**